packages/itwinui-css/src/stepper/stepper.scss

```diff
@@ -17,101 +15,91 @@
 }
 
 @mixin iui-stepper-step {
+  --_iui-stepper-step-background-color: var(--iui-color-background-1);
+  --_iui-stepper-step-border-color: var(--iui-color-foreground-positive);
+  --_iui-stepper-step-number-color: var(--iui-color-foreground-positive);
+  --_iui-stepper-step-text-color: var(--iui-color-foreground-positive);
+  --_iui-stepper-step-track-before-color: var(--iui-color-foreground-positive);
+  --_iui-stepper-step-track-after-color: var(--iui-color-foreground-positive);
+
   flex: 1;
   align-items: center;
   display: flex;
   flex-direction: column;
   @include iui-text(small);
-  color: var(--iui-color-background-positive);
+  color: var(--_iui-stepper-step-text-color);
 
-  // Completed steps
-  .iui-stepper-circle {
-    align-items: center;
-    border-radius: 50%;
-    display: flex;
-    width: $iui-l;
-    height: $iui-l;
-    justify-content: center;
-    overflow-wrap: break-word;
-    user-select: none;
-    transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out,
-      color $iui-speed-fast ease-out;
-    border: 1px solid var(--iui-color-foreground-positive);
-    background-color: var(--iui-color-background-1);
-    color: var(--iui-color-foreground-positive);
+  &:first-of-type {
+    --_iui-stepper-step-track-before-color: transparent;
   }
 
+  &:last-of-type {
+    --_iui-stepper-step-track-after-color: transparent;
+  }
+}
+
+@mixin iui-stepper-step-modifiers {
```

Comment:
This was done to keep `@mixin iui-stepper-step` clean so I could reuse only the necessary parts in `workflow-diagram.scss`.